slides/history_digital_money.tex

```diff
@@ -0,0 +1,299 @@
+\begin{columns}
+\column{0.4\textwidth}
+\begin{figure}
+	\begin{tikzpicture}[scale=1]
+		\input{../assets/figures/before_bitcoin}
+		\filldraw[draw=black, fill = focus, thick] (0, -0.5 cm) circle (2pt);	
+	\end{tikzpicture}
+\end{figure}
+\column{0.7\textwidth}
+	\textbf{Reusable Proofs of Work - Hal Finney} \cite{finney2005}
+	\vspace{0.5em}
+	\begin{small}
+	\begin{itemize}
+		\item Combines ideas of Wei Dai and Adam Back.
+		\item A RPoW client can create a RPoW token by providing a proof-of-work string and signing with his private key.
+		\item Server can map that token to the signing key.
+		\item Client can give the token to another key by signing a transfer order to a public key.
+		\item Server registers the token as belonging to the corresponding private key.
+	\end{itemize}
+	\end{small}
+\end{columns}
+\end{frame}
+%%%	
+
+
+%%%
+\begin{frame}{Before Bitcoin}
+\begin{columns}
+\column{0.4\textwidth}
+\begin{figure}
+	\begin{tikzpicture}[scale=1]
+		\input{../assets/figures/before_bitcoin}
+		\filldraw[draw=black, fill = focus, thick] (0, -0.5 cm) circle (2pt);
+	\end{tikzpicture}
+\end{figure}
+\column{0.7\textwidth}
+	\textbf{Bit Gold - Nick Szabo}
+	\vspace{0.5em}
+	\begin{small}
+	\begin{itemize}
+		\item Describes the combination of the proof-of-work algorithm for competitive money creation.
+		\item Computing power is used to collateralise a public ledger, which grows into a chain of blocks through a reflexive reference.
+		\item "Thus, it would be very nice if there were a protocol whereby unforgeably costly bits could be created online with minimal dependence on trusted third parties, and then securely stored, transferred, and assayed with similar minimal trust. Bit gold." \cite{szabo2005}
+	\end{itemize}
+	\end{small}
+\end{columns}
+\end{frame}
+%%%	
+
+
+%%%
+\begin{frame}{After Bitcoin: Forks and Altcoins}
+	\begin{columns}
+		\column{0.4\linewidth}
+		\begin{figure}
+			\begin{tikzpicture}[scale=1]
+				\input{../assets/figures/after_bitcoin}
+				\filldraw[draw=black, fill = focus, thick] (0, 3.5 cm) circle (2pt);
+				\filldraw[draw=black, fill = focus, thick] (0, 1.5 cm) circle (2pt);
+			\end{tikzpicture}
+		\end{figure}
+		\column{0.6\linewidth}
+			\begin{tikzpicture}
+				\node (Dogecoin) {\includegraphics[scale=0.13]{../assets/images/doge}};
+				\node[right =1mm of Dogecoin] {\textbf{Dogecoin}};
+			\end{tikzpicture}
+		\begin{small}
+			\begin{itemize}
+			\item Created as a fun cryptocurrency based on the popular "doge" internet meme.
+			\item Block time of 1 min and uncapped supply.
+			\item Can be mined on Windows, Mac, Linux or with a GPU.
+			\end{itemize}
+		\end{small}
+		\vspace{0.5em}
+			\begin{tikzpicture}
+				\node (Zcash) {\includegraphics[scale=0.08]{../assets/images/zcash}};
+				\node[right =1mm of Zcash] {\textbf{Zcash}};
+			\end{tikzpicture}
+		\begin{small}
+			\begin{itemize}
+				\item Focused on privacy and anonymity.
+				\item Uses zk-SNARK zero-knowledge proof technology.
```

Comment:
We probably should not repeat "zero knowledge" since zk-SNARK = Zero-Knowledge Succinct Non-Interactive Argument of Knowledge. 
```suggestion
				\item Uses zk-SNARK
```